### PR TITLE
testapi: Slightly simplify upload_logs

### PR DESCRIPTION
**Pull request by okurz**
* #2553

### CODE REVIEW

**Review by Martchus — APPROVED** [review]

File: testapi.pm [hunk 1]
```diff
@@ -2189,12 +2189,8 @@ sub upload_asset ($file, $public = undef, $nocheck = undef) {
...
     my $basename = basename($file);
     $cmd .= autoinst_url("/upload_asset/$basename");
-    if ($nocheck) {
-        type_string("$cmd\n");
-    }
-    else {
-        return assert_script_run($cmd);
-    }
+    return assert_script_run($cmd) unless $nocheck;
+    type_string("$cmd\n");
```
Comment (kalikiana) [inline testapi.pm:2193]: Please get #2553 done first as that will add the test coverage for this

**Review by kalikiana — CHANGES_REQUESTED** [review]

File: testapi.pm (hunk 1 above)
Comment (okurz) [reply testapi.pm:2193]: The idea was that with this change adding test coverage is easier. That's what we discussed today in the daily

**Review by perlpunk — COMMENTED** [review]
Subject talks about `upload_logs`, but change is in `upload_asset`

File: testapi.pm (hunk 1 above)
Comment (kalikiana) [reply testapi.pm:2193]: Why not just add it to the other branch? Making more untested changes seems counter-intuitive to me

Comment (okurz) [reply testapi.pm:2193]: Well, either we merge here first and @asdil12 rebases or they pick my commit into their branch

**Comment by okurz** [comment]
@mergifyio rebase

**Pull request closed without merging**